packages/frontend/core/src/bootstrap/polyfill/browser.ts

```diff
@@ -4,14 +4,8 @@ import { polyfillPromise } from './promise-with-resolvers';
 import { polyfillEventLoop } from './request-idle-callback';
 import { polyfillResizeObserver } from './resize-observer';
 
-export function polyfillElectron() {
-  polyfillResizeObserver();
-}
-
-export async function polyfillBrowser() {
-  polyfillResizeObserver();
-  polyfillEventLoop();
-  await polyfillI18n();
-  await polyfillPromise();
-  await polyfillDispose();
-}
+polyfillResizeObserver();
+polyfillEventLoop();
+await polyfillI18n();
+await polyfillPromise();
+await polyfillDispose();
```

Comment:
I recalled that top-level await caused some issues before but I forget the exact reasons

Reply:
top level await comes for a long time in apps/setup